src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/talismans/MagicianTalisman.java

```diff
@@ -90,4 +95,14 @@ private boolean hasConflicts(Set<Enchantment> enchantments, TalismanEnchantment
         return true;
     }
 
+    /**
+     * This method checks when enchanting book
+     * can it be applied a extra {@link Enchantment} or not.
+     *
+     * @return Whether the book can be apply extra {@link Enchantment}
+     */
+    @Nonnull
+    public ItemSetting<Boolean> isAllowEnchantBook() {
+        return allowEnchantBook;
```

Comment:
Simply return the boolean here, no point in making a getter for the itemsetting.
```suggestion
    public boolean isEnchantmentBookAllowed() {
        return allowEnchantmentBooks.getValue();
```